README.md

```diff
@@ -80,16 +141,349 @@ To create the virtual environment:
+the `shape_main.cxx` application. They should name the executable created as:
+`<product_name>_shape_main_linux` and compress it into a `.zip.`
+
+Example:
+
+![zip](./doc/img10.png)
+
+Then they should upload the executable to git in the following way:
+
+> **Note**: to do it with the command line see `gh release`
+
+1. In the main page, go to *Releases*.
+
+![Releases](./doc/img11.png)
+
+2. Inside the latest release, go to *Edit*.
+
+![Edit](./doc/img12.png)
+
+3. Upload the executables in *Attach binaries by dropping them here or
+selecting them* (leave all the other fields as they were).
+Keep marked *Set as the latest release*, and press *Update release*.
+
+![Attach](./doc/img13.png)
+
+> **Note**: when a new executable is uploaded we should generate a new report.
+To do it, go to
+[Automation with GitHub Actions](#automation-with-github-actions)
+## Create a new release
+
+### When to create a release
+
+There are two cases when we will create a new release:
+* Once a year \
+    At the beginning of a new year we will create a new release.
+    * Before creating the release we should:
+        * Generate a report (see
+          [Automation with GitHub Actions](#automation-with-github-actions)).
+        * Upload the report generated as an asset to the latest release
+          (see
+          [Upload executable](#upload-executable)
+          and follow the same steps but with the report
+          instead than with the executable).
+* When there is a new functionality in `shape_main.cxx`,
+ `interoperability_report.py`, or the creation of any other file that
+ could be considered as a new functionality.
+
+Every time we create a new release we need to select which executables
+we want to maintain in the new release. This new release should contain
+the last version uploaded for every vendor.
+The executables could also be uploaded (or removed) later.
+
+### Release and tag name
+
+The name of the release and the tag associated with it will be the same.
+It should follow the next rule: \
+`vx.y.z`
+* x: major version
+* y: minor version
+* z: year
+
+Depending on the importance of the changes from the old release to the new one,
+it will be a major or a minor version change:
+* New functionality, major change -> major version
+* Bug fix or new functionality, minor change -> minor version
+
+
+> Case of use: actual name -> `v1.0.2022`
+> * If there is a small change in the repository: `v1.1.2022`
+> * If there is a big change in the repository: `v2.0.2022`
+> * At the beginning of a new year: `v1.0.2023`
+>   * If last release was `v1.2.2022`, it would be `v1.2.2023`
+
+### Process of creating the release
+
+It is important that we set the release that we are creating as the
+latest release, and that we do not create it as a pre-release.
+Here it is explained how to create the release with the graphic interface.
+
+> **Note**: to do it with the command line see `gh release` \
```

Comment:
```suggestion
> **Note**: to do it with the command line see `gh release`
```

Reply:
I put it so it was written in two lines (maybe it is easier to read)